xarray/convert.py

```diff
@@ -8,8 +8,10 @@
 
 from .core.dataarray import DataArray
 from .core.pycompat import OrderedDict, range
+from .core.dtypes import get_fill_value
 from .conventions import (
     maybe_encode_timedelta, maybe_encode_datetime, decode_cf)
+from dask.array import ma
```

Comment:
This needs to be guarded inside a function, as dask is still not a hard requirement for xarray.